v2/fb-xcom-api-validator/tests/setup/preload-capabilities.ts

```diff
@@ -0,0 +1,9 @@
+import 'ts-node/register';
+import config from '../../src/config';
+
+async function preloadCapabilities() {
+  const capabilitiesConfig = config.get('capabilities');
+  global['capabilities'] = capabilitiesConfig;
+}
```

Comment:
Why though? The capabilities are already in the config. What do we gain by having them in a global variable? In general, we should minimize our usage of Jest's `setupFiles`.